iam_python_sdk/client.py

```diff
@@ -393,14 +393,16 @@ def _apply_user_permission_resource_values(self, grantedPermissions: List[Permis
         Returns:
             List[Permission]: List of permission with applied user permission
         """
+        result = copy.deepcopy(grantedPermissions)
```

Comment:
Please rename `result` variable to `granted_permissions`

Reply:
sure!